### ISSUE TITLE
Kuroshiro is not a constructor

### ISSUE THREAD

**Issue by jwalk00** [hexenq/kuroshiro#85]
I cannot for the life of me get this to work. The best result I can achieve is "Kuroshiro is not a constructor".

I have added the two min js files to my project folder, and enqueue them:
```
<script src="js/kuroshiro.min.js"></script>
<script src="js/kuroshiro-analyzer-kuromoji.min.js"></script>
```
Then added my own script:
`<script type="module" src="js/ks.js"></script>`

which contains:
```
var kuroshiro = new Kuroshiro();
await kuroshiro.init(new KuromojiAnalyzer());
var result = await kuroshiro.convert("感じ取れたら手を繋ごう、重なるのは人生のライン and レミリア最高！", {mode:"furigana", to:"hiragana"});
console.log(result);
```

Then I get: 
Uncaught TypeError: Kuroshiro is not a constructor
    at ks.js:2
    
Does Kuroshiro require the use of webpack/gulp or a Javascript frame work like React/Vue? I'm trying to do it with just base html/css/js files and I don't know if that's just plain wrong...

**Comment by phamhuuan** [comment]
@jwalk00 try this https://github.com/hexenq/kuroshiro/issues/66#issuecomment-942162775

**Comment by jwalk00** [comment]
> @jwalk00 try this [#66 (comment)](https://github.com/hexenq/kuroshiro/issues/66#issuecomment-942162775)
```
"kuroshiro": "^1.2.0",
"kuroshiro-analyzer-kuromoji": "^1.1.0",
```
in package.json

But again I'm trying to use this without a package manager/bundler. Just enqueueing the scripts in head and running.

**Comment by jwalk00** [comment]
Fixed this by downgrading to v1.1.2 and using those dist files.

Then applying this fix when I got Analyzer errors: https://github.com/hexenq/kuroshiro/issues/27#issuecomment-686428617

**Comment by chinenvinicius** [comment]
isnt the v1.1.2 the latest version?
@jwalk00

**Comment by jwalk00** [comment]
> isnt the v1.1.2 the latest version? @jwalk00

1.2.0 is the latest release https://github.com/hexenq/kuroshiro/releases/tag/1.2.0

**Comment by CQzhangjiawang** [comment]
so, how do you fiexed the problem.

I used the latest 1.2.0 release to `npm install`, `npm run build`, and the code is same as to you.
but the error alwasy exists.

I need help.

Please help me!
@jwalk00

**Comment by jwalk00** [comment]
"Fixed this by downgrading to v1.1.2 and using those dist files."

Use v1.1.2
